BTrees/tests/testBTrees.py

```diff
@@ -114,14 +114,14 @@ def testBasicOps(self):
         t, keys = self._build_degenerate_tree()
         self.assertEqual(len(t), len(keys))
         self.assertEqual(list(t.keys()), keys)
-        # has_key actually returns the depth of a bucket.
```

Comment:
Oh boy, I hope nobody relied on that! :)

Reply:
The doc string says "Return a true value if the key is present, else a false value." while the example says "returns a true value, but exactly what undefined". So I hope not also :)